README.md

```diff
@@ -11,3 +11,154 @@ This Repository
 ===============
 
 This repository contains the markup needed to deploy the floeproject.org website.
+
+
+News Item Template
+==================
+
+News item file naming convention: "YYYY-MM-DD name-name-name.html". Add this file to the news folder.
+
+Note: you will also need to add your news item to the top of the news archive (in news/index.html).
+
+Note: remember to update the floeproject.org page with a short summary of your news item.
+
+The following provides a template for creating a news item html file:
+
+<!DOCTYPE html>
```

Comment:
Instead of including the code in the readme, perhaps it makes more sense to make an news-template.html file with the template content in it. Then update the readme to reference that new file instead.